modules/Emails/EmailsController.php

```diff
@@ -139,19 +139,51 @@ public function action_ComposeView()
      */
     public function action_send()
     {
-        $this->bean = $this->bean->populateBeanFromRequest($this->bean, $_REQUEST);
-        $this->bean->save();
+        global $current_user;
+        global $app_strings;
 
-        $this->bean->handleMultipleFileAttachments();
+        $this->bean = $this->bean->populateBeanFromRequest($this->bean, $_REQUEST);
+        $inboundEmailAccount = new InboundEmail();
+        $inboundEmailAccount->retrieve($_REQUEST['inbound_email_id']);
```

Comment:
It's used by Detail/ListView and when importing and marking emails, but not when sending emails.

It is present in the "compose view" url when you forward or reply to an email. The url param value is blank with the Mozaik editor, but populated with the other 2 editors.

Either way, it doesn't seem to cause any problems. I tested without this PR (but with #3645) and the `inbound_email_record` id was ignored and emails would still be sent through the system account.